Changelog.md

```diff
@@ -5,6 +5,9 @@ SPDX-License-Identifier: GPL-3.0-or-later
 -->
 
 # Changelog
+# 2021/05/XX: Release of v2.3
+- Changed cout output to a Logger which provides different levels. How to set them is shown under the --help option
+- CP in the dark is now added as a model
```

Comment:
@lisabiermann and @JonasMueller1991 : Please change this line so that it includes some reference where the user can read about this model and how it is set up.